### ISSUE TITLE
ray-network is not installed with make install 

### ISSUE THREAD

**Issue by crshrprt** [Houston4444/RaySession#215]
I was trying to use a network session and after a few errors I realized that ray-network is not installed in any suitable $PATH when building RaySession. Once I copied ray-network in a suitable $PATH the wizard appeared and I was able to launch a network session correctly.

**Comment by crshrprt** [comment]
I spoke too early, on a second try, as soon I connect from the network wizard it crashes the slave:

$ ray-daemon -p 1234                                                                                   
[ray-daemon]URL : osc.udp://192.168.1.10:1234/
[ray-daemon]      osc.udp://slave.fritz.box:1234/
[ray-daemon]ROOT: /home/user/Ray Network Sessions
[ray-daemon]GUI connected at osc.udp://192.168.1.12:11753/
[ray-daemon]Attempting to open /home/user/Ray Network Sessions/Net
Traceback (most recent call last):
  File "/usr/local/share/raysession/src/daemon/session_signaled.py", line 165, in osc_receive
    function(path, args, src_addr)
  File "/usr/local/share/raysession/src/daemon/session_signaled.py", line 57, in wrapper
    sess.next_function()
  File "/usr/local/share/raysession/src/daemon/session.py", line 837, in next_function
    next_function(*arguments)
  File "/usr/local/share/raysession/src/daemon/session.py", line 1197, in close_no_save_clients
    self._wait_and_go_to(duration, self.close_no_save_clients_substep1,
  File "/usr/local/share/raysession/src/daemon/session.py", line 736, in _wait_and_go_to
    follow()
  File "/usr/local/share/raysession/src/daemon/session.py", line 1217, in close_no_save_clients_substep1
    self._wait_and_go_to(120000, self.next_function, ray.WaitFor.QUIT, True)
  File "/usr/local/share/raysession/src/daemon/session.py", line 736, in _wait_and_go_to
    follow()
  File "/usr/local/share/raysession/src/daemon/session.py", line 837, in next_function
    next_function(*arguments)
  File "/usr/local/share/raysession/src/daemon/session.py", line 1832, in preload
    self.desktops_memory.read_xml(c)
                                  ^
UnboundLocalError: cannot access local variable 'c' where it is not associated with a value
Annullato

**Comment by crshrprt** [comment]
Deleted the session from the slave, now it's working again. The problem seems only the missing ray-network on installation for now.

**Comment by Houston4444** [comment]
thanks for your code report ! *Desktops memory* option was broken. Also, loading a session with 0 client made the daemon crash, it's fixed now.

However, I don't see any problem with the ray-network executable, here it works, even when RaySession is installed.
Note that the `src/bin` folder (in RaySession source code) is added a top of $PATH at raysession and ray-daemon startup, this way, no need to install many executables, and it warranties that the GUI launches the daemon with the same version (and not another one present in the system).

**Comment by crshrprt** [comment]
Indeed, the src/bin is correctly added to the $PATH, my mistake. Thank you for the clue, upon further investigation I discovered that for some reason the binaries in src/bin had the executable permission only for root. Don't know how it happened (I vaguely remember some problem with make install not setting the right permissions at some point in time, can't remember the details). However, I checked on other machines and also a new installation on another machine and ray-network works correctly. Sorry for the noise, at least was useful for fixing other things. :)
